### ISSUE TITLE
Option to disable youtube-dl

### ISSUE THREAD

**Issue by normen** [shiningw/ncdownloader#119]
**Is your feature request related to a problem? Please describe.**
I am running NextCloud in a minimal php-fpm container with no python or ffmpeg installed and this plugin is spewing out errors because of that.

**Describe the solution you'd like**
I'd like an option in the settings to disable the youtube-dl functionality altogether.

**Describe alternatives you've considered**
I could install ffmpeg and python in my container just to avoid this error but that doesn't seem feasible.

**Additional context**
Youtube-dl functionality is great but I am already using peertube for my video needs as NextCloud isn't really suited for that.

Thanks for the attention!

**Comment by shiningw** [comment]
Hi,
there is an option to disable such error display in the bottom left sidebar.

**Comment by normen** [comment]
Right, thanks, that doesn't disable the youtube-dl button however, which is confusing for users.
